tajo-core/src/main/java/org/apache/tajo/util/history/HistoryWriter.java

```diff
@@ -49,8 +56,10 @@
   public static final String QUERY_LIST = "query-list";
   public static final String QUERY_DETAIL = "query-detail";
   public static final String HISTORY_FILE_POSTFIX = ".hist";
+  private static short REPLICATION = 1;
```

Comment:
Since history may be not critical data in many cases, one replication would be enough. But, in some cases, it can be very important. How about improve it to be some configurable?